studies/india_districts/lockdown_evaluation.py

```diff
@@ -1,6 +1,7 @@
 from itertools import product
 from pathlib import Path
 from typing import Dict, Optional, Sequence
+from datetime import datetime
```

Comment:
don't think we need this import any more?

Reply:
deleted